docs/content/latest/yugabyte-cloud/cloud-network/vpc-peers/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: VPC Peering
+headerTitle: VPC Peering
+linkTitle: VPC Peering
+description: Configure VPC peering.
+image: /images/section_icons/secure/tls-encryption/connect-to-cluster.png
+headcontent: Set up VPC peering for your Yugabyte Cloud clusters.
+menu:
+  latest:
+    identifier: vpc-peers
+    parent: cloud-network
+    weight: 400
+---
+
+Virtual Private Cloud (VPC) peering allows applications running on instances on the same cloud provider as your Yugabyte Cloud clusters to communicate with those clusters without traversing the public internet; all traffic stays within the cloud provider's network. Yugabyte Cloud supports AWC and GCP for self-managed peering. For Azure, contact Yugabyte Cloud support.
+
+To set up peering in Yugabyte Cloud, you first create a VPC to host your clusters, then peer it with the VPC that hosts client applications. Once the peering connection is active, you can deploy a cluster in the VPC you created. Any cluster deployed using that VPC will then be able to connect with any application in the application VPC.
+
+{{< note title="Note" >}}
+
+To use VPC peering in a cluster, your cluster must be deployed in a dedicated VPC that is peered with your application VPC. You must set up a dedicated VPC and peering before deploying your cluster. VPC peering is not supported in free clusters.
+
+{{< /note >}}
+
+## VPC peering
+
+Setting up and peering a VPC works as follows:
+
+- [Create a VPC](#create-a-vpc)
+- [Create a Peering Connection](#create-a-peering-connection)
+- [Configure the provider](#configure-the-cloud-provider)
+
+Once the peering connection is active, you can deploy clusters in the VPC. Refer to [Create clusters](../../cloud-basics/create-clusters/).
```

Comment:
The clusters can actually be created before the peering is active as well. Maybe this can be under create a VPC. Also, we can change this to be like once the peering connection is active, you can connect to your cluster deployed in the VPC with the peering connection?